src/DynamoCore/DynamoCore.csproj

```diff
@@ -23,14 +23,15 @@
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
     <PackageReference Include="Microsoft.Win32.Registry" Version="4.7.0" />
     <PackageReference Include="System.Drawing.Common" Version="6.0.0" GeneratePathProperty="true" ExcludeAssets="runtime" />
+    <PackageReference Include="DynamoVisualProgramming.Analytics" Version="4.0.0.1432" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">         
     <Reference Include="System.Configuration" />
     <Reference Include="Microsoft.CSharp" />
+    <PackageReference Include="DynamoVisualProgramming.Analytics" Version="3.0.1.740" />
```

Comment:
do we want to update analytics for net48 ?
I do not see why we net48 should lag behind with the analytics version. Unless it would break something...
any ideas @QilongTang @aparajit-pratap @mjkkirschner ?

Reply:
Hi @pinzart Bit sure if you want to expand the scope. I think @aparajit-pratap previously already removed all the GA stuff in Analytics package so if we update the net48 version, then we probably want to remove all the mention of GA in Dynamo as well to pair. Our team does have a task in our backlog about it

Reply:
@pinzart90 yes! the update version breaks something in this code 
<img width="561" alt="image" src="https://github.com/DynamoDS/Dynamo/assets/134449953/52dfb685-a006-40c8-a728-2a41c13c4fdf">
Here I'm in the net 6 solution, but in net48 with updated version, it is not recognized the Autodesk.Analytics. Maybe I can remove this code?

Reply:
@aparajit-pratap so we removed GA from the analytics package even for net48 ?
If yes, dot we want to remove it in Dynamo too  (on net48) ? If not...we're stuck with the old version of analytics.

Reply:
I think we do as I mentioned we have a task, but if your team is up for it, we can do it here..

Reply:
There's just one analytics package for both net48 and net6.0, which is the latest - 4.0.0.1432.

Reply:
@aparajit-pratap using 4.0.0.1432 in net48 breaks code:
<img width="675" alt="image" src="https://github.com/DynamoDS/Dynamo/assets/134449953/373c1381-ee5b-4eec-af06-3a5d5d43104e">

Reply:
@aparajit-pratap shouldn't the google references be removed from DynamoCore in both net48 and net6 builds?

Reply:
@pinzart I think this answers your question:
<img width="407" alt="Screenshot 2023-07-05 at 2 43 23 PM" src="https://github.com/DynamoDS/Dynamo/assets/508936/85a9fbf9-f645-43ca-b812-fa37755e55b3">

Reply:
ok, then the question is if we want to clean up the google APIs on net48 too

Reply:
Wasn't EOL for GA end of June?

Reply:
Sorry for the late response. Yes, we should remove all GA references from Core in this PR itself so we can use the latest analytics package (4.x) even for net48.

Reply:
@aparajit-pratap @mjkkirschner should we remove all references to GoogleAnalytics (including preferences settings, UI etc) ?

Reply:
Either remove or make them do nothing for now.